### ISSUE TITLE
[High Contrast] SeZo button has no hover visual

### ISSUE THREAD

**Issue by rigdern** [winjs/winjs#1398]
1. Open sezo.html from http://1drv.ms/1hOG9RG in Chrome
2. Put the machine into High Contrast 1
3. With the mouse, hover over the Semantic Zoom's zoom out button in the bottom right corner of the ListView

**Unexpected result:** Zoom out button doesn't have a hover visual -- it looks the same as its rest visual.

![image](https://cloud.githubusercontent.com/assets/199935/9398696/d8de7fd2-475e-11e5-9afe-aa90748dc6c3.png)
